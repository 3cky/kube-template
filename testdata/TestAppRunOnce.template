the quick brown fox jumps over the lazy dog

pods: {{range pods}}{{.Name}}{{end}}
