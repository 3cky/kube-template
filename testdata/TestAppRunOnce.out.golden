the quick brown fox jumps over the lazy dog

pods: pod1
